### DISCUSSION TITLE
**EPPR** Q&A | Clase 303

### DISCUSSION THREAD

**Discussion by recurs1v0** [EPPR/kardex#22] [General]
## 👋 Welcome!
  Utiliza este foro para :
  * Preguntar tus dudas
  * Compartir ideas
  * Pegar links de interés común
  * Ayudar a tus compañeros y crear una comunidad más fuerte 🤝.

**Comment by eppr-lxps** [comment]
@MSuaz 

Hola Tai, muy buena [tarea](https://github.com/MSuaz/MSuaz.github.io/blob/main/html/index.html)

Por favor revisa los ISSUES de tu repo para acomodar el código como lo vimos en clase.
